goss/goss-mysql.yaml

```diff
@@ -0,0 +1,20 @@
+service:
+  mysql:
+    enabled: true
+    running: true
+user:
+  mysql:
+    exists: true
+    uid: 999
+    gid: 999
+    groups:
+    - mysql
+    home: /home/mysql
+    shell: ""
+group:
+  mysql:
+    exists: true
+    gid: 999
+process:
+  mysql:
+    running: true
```

Comment:
I think the process name in this case is `mysqld`. Also, would it make sense to add a section that waits for a listener on the port 3306?

e.g. 

```
tcp:3306:
  listening: true
  ip:
    - 0.0.0.0
```